.github/workflows/docs.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   ocaml:
     name: Docs
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Why not `ubuntu-latest` here?

Reply:
testing and building the toolstack is more fragile with regards to distribution updates, this is because the official opam repository protects us from unstability in the packages they maintain (like ocamlformat), but not our own.

For example to update to ubuntu 22.04 we had to change the python testing, because python 2 is simply not available.